components/dropindicator/index.css

```diff
@@ -10,54 +10,89 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+.spectrum-DropIndicator {
+  --spectrum-dropindicator-circle-border-color: var(--spectrum-blue-800);
+  --spectrum-dropindicator-border-color: var(--spectrum-blue-800);
+  --spectrum-dropindicator-border-size: var(--spectrum-border-width-200);
+  --spectrum-dropindicator-circle-size: 12px;
+}
+
 .spectrum-DropIndicator {
   position: relative;
+  background: var(
```

Comment:
Does it make sense/Is it possible to create local-only variables so that we don't have to duplicate this mod-logic? Like instead of everywhere doing a `var(--mod, var(--spectrum))` thing couldn't I just do like:

```
.spectrum-DropIndicator {
 --local-dropindicator-border-color: var(--mod, var(--spectrum));
}
```

Or would that create an unacceptable amount of css custom properties?

Reply:
I am not sure how this might affect how we have been using custom properties, especially what we've been doing with passing through custom properties. Would be curious on @jawinns thoughts.

Reply:
We do use local variables in some cases to simplify some state-based changes or calculated values. So far we haven't used them for the purposes of avoiding repeating the --mod within the list of fallbacks within a var() function. Westbrook had asked a similar question in a PR (square tooltips), and after discussing with the team it sounded like we're not planning to use that pattern right now. I think we want to avoid the additional variables and additional layer of abstraction it would create---IMO I think that's better though I understand the "D.R.Y." thoughts for it in some cases.